style.css

```diff
@@ -57,3 +67,39 @@ nav {
     background: url('https://www.powerhouseenergy.net/wp-content/uploads/2016/10/investment-banner.jpg') top left/cover no-repeat; 
     height: 300px;
   }
+
+  .flexbox {
+    position: relative;
+    width:1000px;
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  .flexbox .box1 {
+    position: relative;
+    max-width: 450px;
+    margin: 10px;
+    padding: 15px;
+    display:flex;
+  }
+
+  .flexbox .box1 .img1 {
+    width: 100px;
+    flex: 0 0 100px;
+  }
+ .flex .box1 .img1 img {
```

Comment:
the last `img` is saying apply this to ALL img tags. its probably not what you want :)

Reply:
Thank you, Joel for you comments! After your comment, I changed the search bar position and tried to delete the img styling which u've mentioned here.  so for now it's deleted. 
And year about name of classes, I don't know how to name them, but for future I'll keep in mind!